cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.CharField(max_length=100, blank=True, default="")
```

Comment:
The `description` field has a `max_length` of 100 here, but in the migration file, it is set to 255. Ensure that both the model and the migration file have the same `max_length` to avoid inconsistencies.